infrastructure/module/eks3/helm_release.tf

```diff
@@ -0,0 +1,36 @@
+resource "helm_release" "alb-controller" {
+  name       = "aws-load-balancer-controller"
+  repository = "https://aws.github.io/eks-charts"
+  chart      = "aws-load-balancer-controller"
+  namespace  = "kube-system"
```

Comment:
Create new namespace just for LB controller and create it with:
**create_namespace = true**